lua/core/screen.lua

```diff
@@ -184,13 +204,19 @@ Screen.text_center = function(str) _norns.screen_text_center(str) end
 -- @tparam number degrees : degress to rotate
 Screen.text_center_rotate = function(x, y, str, degrees) _norns.screen_text_center_rotate(x, y, str, degrees) end
 
---- calculate width of text.
--- uses currently selected font.
+--- calculate width of text given current font and draw state.
+-- completes asynchronousely via system callback
+-- (FIXME: see screen.handle_text_extents() or whatever)
 -- @tparam string str : text to calculate width of
-Screen.text_extents = function(str) return _norns.screen_text_extents(str) end
+Screen.text_extents = function(str) _norns.screen_text_extents(str) end
+
+-- get the current drawing position in the screen surface
+-- @treturn number x
+-- @treturn number y
+Screen.current_point() = function() return _norns.screen_current_point() end
```

Comment:
typo here that prevents startup (parens before =)

Screen.current_point = function() ...

Reply:
Oops! Thought I had addressed that after the merge -- one second.